packages/components/menu/src/context-menu.tsx

```diff
@@ -24,3 +23,6 @@ export const ContextMenu: FC<ContextMenuProps> = (props) => {
     </ContextMenuProvider>
   )
 }
+
+ContextMenu.displayName = "ContextMenu"
+ContextMenu.__ui__ = "ContextMenu"
```

Comment:
There’s no need to specify displayName for components that don't use forwardRef.

Reply:
@hirotomoyamada 
> There’s no need to specify displayName for components that don't use forwardRef.

Is this right?

Reply:
@koralle 

Yes, correct.

However, there are unnecessary `displayName` everywhere in the Yamada UI, so I would like to revamp it somewhere.